src/Features/Core/Features.csproj

```diff
@@ -537,7 +537,6 @@
   <ItemGroup>
     <Compile Include="CodeFixes\Suppression\SuppressionCodeAction.cs" />
     <None Include="packages.config" />
-    <PublicAPI Include="PublicAPI.txt" />
```

Comment:
Yes this is not intended. Probably an accidental delete on my part of another merge issue. I'll revert this.